USER_GUIDE.md

```diff
@@ -420,6 +425,66 @@ query = {
 response = client.plugins.alerting.acknowledge_alert(query)
 print(response)
 ```
+
+### Security plugin
+
+#### **Creating a role**
```

Comment:
I think another PR was getting rid of these `**` bolding, let's remove those everywhere?